### ISSUE TITLE
Staging Review finding: File input error triggered while file is uploading

### ISSUE THREAD

**Issue by shiragoodman** [department-of-veterans-affairs/va.gov-team#114648]
_Need help? Please review [how to read a Staging Review ticket](https://depo-platform-documentation.scrollhelp.site/collaboration-cycle/anatomy-of-a-staging-review-issue-ticket). Tag `@platform-governance-team-members` on Slack if you need further assistance._

## Product Information
**Team:** Accredited Rep Facing
**Product:** Accredited Rep Portal
**Feature:** Claims Form Upload

## Findings details
**VA.gov Experience Standard - issue:** [User encounters accurate, consistent content.](https://design.va.gov/about/experience-standards/#user-encounters-accurate-consistent-content)
**VA.gov Experience Standard - category:** [Consistency](https://design.va.gov/about/experience-standards/#category-consistency)
**Launch-blocking:** No
**Design System review:** No
**Collab Cycle Reviewer:** @rsmithadhoc (Accessibility)

### Description

Right after selecting a file to upload, it shows "Uploading..." as expected, but it also shows an error message above it, which could be confusing.

<details><summary>Link, screenshot or steps to recreate</summary>

https://staging.va.gov/representative/representative-form-upload/21-686c/upload

<img width="721" height="353" alt="Image" src="https://github.com/user-attachments/assets/05926b2f-73fb-47cd-8e00-2f113d39013d" />

</details>

## Recommended action

Don't trigger the error after a file has been selected and is uploading.

## References

- **Accessibility Defect Severity:** [3: Medium. Should be fixed in 1-3 sprints post-launch.](https://depo-platform-documentation.scrollhelp.site/developer-docs/accessibility-defect-severity-rubric#Accessibilitydefectseverityrubric-a11y-defect-3)
- **WCAG Success Criteria:** [WCAG 4.1.3 Status Messages - Level AA](https://www.w3.org/TR/WCAG22/#status-messages)
- **Modality:** N/A
- **Design System Component:** N/A
- **Design System Pattern or Template:** N/A
- **Design System Foundation:** N/A
- **Content Style Guide:** N/A
- **Context:** N/A

---

### Next Steps for the VFS Team

_How did this touchpoint go? [Give feedback on the Collaboration Cycle](https://dj540s05.optimalworkshop.com/questions/f54355271969f160ef41182ba9ceb61c) at any time._
- [ ] **Questions?** For the most timely response, comment on Slack in your team channel tagging `@platform-governance-team-members` with any questions or to get help validating the issue.
- [ ] **Ticket ownership:** If you believe that this issue is out of scope, not your team's responsibility, or a Design System issue, comment and tag `@platform-governance-team-members` on your team channel in Slack to provide an explanation and who you believe is responsible. The Governance team will follow up.
- [ ] **Close this ticket** when the issue has been resolved or validated by your Product Owner.

**Comment by patrick-brown-oddball** [comment]
We brought this up a bit ago. This is a known issue for Veteran facing forms- https://dsva.slack.com/archives/C044AGZFG2W/p1751377430186189

**Comment by candilemoine** [comment]
@patrick-brown-oddball attach this ticket to the existing Forms issue ticket

**Comment by candilemoine** [comment]
PR was approved but [branch is currently locked by Platform](https://dsva.slack.com/archives/CE4304QPK/p1762359580724189)
